asmcomp/amd64/emit.mlp

```diff
@@ -1110,6 +1111,9 @@ let end_assembly() =
       efa_string = (fun s -> D.bytes (s ^ "\000"))
     };
 
+  let frametable = Compilenv.make_symbol (Some "frametable") in
+  D.size frametable (ConstSub (ConstThis, ConstLabel frametable));
```

Comment:
Where is this label used? Is there any place in the runtime where this piece of information is read?

Reply:
This size information is not read by the runtime.

This exports an ELF size, so that standard ELF binary tools (e.g. `objdump`, `readelf`, etc.) can be used to measure the size of a frame table. (Before this change, such tools show the frametable as having size 0). This information is already exported for OCaml functions, and for assembly stubs since #8708.